### PR TITLE
Add vssTimeout backup configuration option

### PR DESCRIPTION
**Pull request by plasticrake**
Call `duplicacy backup` with `-vss-timeout` option from gilbertchen/duplicacy@be2856ebbd0a90092a775a34b36ff52340e4b379

### CODE REVIEW

**Comment by plasticrake** [comment]
Thanks for creating this utility! It has really helped me getting started using duplicacy. I tried my best to infer how'd you liked to name things and code style, but let me know if you'd like me to change anything for the code to fit better.

I tested running duplicacy-util with:
  1. YAML: vssTimeout not specified
    * Correctly ran without -vss-timeout
  2. YAML: vssTimeout set to 0
    * Correctly ran without -vss-timeout
  3. YAML: vssTimeout set to 180
    * Correctly ran with -vss-timeout 180

Note that internally `duplicacy` will always set to this a minimum of 60 seconds, although I didn't see that documented anywhere:

https://github.com/gilbertchen/duplicacy/blob/0af74616b72391958393dd65ad341e8cbd8f82f7/src/duplicacy_shadowcopy_darwin.go#L134

https://github.com/gilbertchen/duplicacy/blob/be2856ebbd0a90092a775a34b36ff52340e4b379/src/duplicacy_shadowcopy_windows.go#L362

File: duplicacy-util.go [hunk 1]
```diff
@@ -419,8 +419,14 @@ func performBackup() error {
...
-				logMessage(logger, fmt.Sprint("Backing up to storage ", backupInfo["name"],
-					" -vss with ", backupInfo["threads"], " threads"))
+				if backupInfo["vssTimeout"] != "" {
+					cmdArgs = append(cmdArgs, "-vss-timeout", backupInfo["vssTimeout"])
+					logMessage(logger, fmt.Sprint("Backing up to storage ", backupInfo["name"],
+						" -vss -vss-timeout ", backupInfo["vssTimeout"],  " with ", backupInfo["threads"], " threads"))
+				} else {
+					logMessage(logger, fmt.Sprint("Backing up to storage ", backupInfo["name"],
+						" -vss with ", backupInfo["threads"], " threads"))
+				}
```
Comment (jeffaco) [inline duplicacy-util.go]: This was a touch sloppy when I implemented it, and now it really shows. I see that you continued what I had done, which is great, but not when what I had done kinda sucks ... :smile:

Excuse typos, not checking for syntax. Could you change this to be something like:

```
vssFlags := ""
if backupInfo["vss"] == "true" {
    cmdArgs = append(cmdArgs, "-vss")
    vssFlags = " -vss"
   if backupInfo["vssTimeout"] != "" {
    cmdArgs = append(cmdArgs ...)
    vssFlags := fmt.Sprintf("%s -vss-timeout %s", vssFlags, backupInfo[vssTimeout"]
  }
```

Now you can have one LogMessage for "Backup up to storage", which I think is cleaner.

File: duplicacy-util.go [hunk 2]
```diff
@@ -413,17 +413,17 @@ func performBackup() error {
 
 	// Perform backup/copy operations if requested
 	if cmdBackup {
-		for i := range configFile.backupInfo {
+		for _, backupInfo := range configFile.backupInfo {
```
Comment (jeffaco) [inline duplicacy-util.go]: I like what you've done here. Two things to be certain of:

1. Will order of items in the range be identical if the range has multiple items (not certain, I'd need to check to be sure, but it's probably the same), and

2. In the same section, for `copy`, `prune`, and `check` operations, the old construct was used. Can you change those constructs as well?

I'm also picky about commits. I'm fine with two commits to contain all this, but review comments should be incorporated into existing commits.

Comment (plasticrake) [reply duplicacy-util.go]: The range should be iterating the same as before, I'm just capturing the value and discarding the index, `backupInfo` should be equivalent to the old `configFile.backupInfo[i]` in this case. I can update the other `for` loops to match

**Comment by jeffaco** [comment]
Changes merged with master, thanks!

**Pull request closed without merging**